.github/workflows/lte-integ-test.yml

```diff
@@ -45,6 +45,7 @@ jobs:
           sudo mkdir -p /etc/vbox/
           sudo touch /etc/vbox/networks.conf
           sudo sh -c "echo '* 192.168.0.0/16' > /etc/vbox/networks.conf"
+          sudo sh -c "echo '* 3001::/64' >> /etc/vbox/networks.conf"
```

Comment:
I assume this is also needed in `bazel-cache-push.yml` and `build_all.yml`. `cwf-integ-test.yml` uses another vagrant file - imo it's not needed there.

Reply:
Is there a way to verify once I make changes in bazel-cache-push.yml and build_all.yml files? Currently CI checks are passing for bazel build and build_all

Reply:
`bazel-cache-push.yml` and the relevant parts of `build_all.yml` only run on master merges. you could run the jobs on your fork - I expect them to fail because secrets are needed here. But you should see that the vms are spinning up. On the other hand I am quite confident that this change will work on these workflows if it worked for the lte integ tests.